sugoi-api-ldap-store-provider/src/main/java/fr/insee/sugoi/store/ldap/LdapReaderStore.java

```diff
@@ -318,14 +319,15 @@ private SearchResultEntry getEntryByDn(String dn) {
       LdapMapper<ResultType> mapper)
       throws LDAPSearchException {
     SearchRequest searchRequest = new SearchRequest(baseDn, scope, filter, "*", "+");
-    LdapUtils.setRequestControls(searchRequest, pageableResult);
-    SearchResult searchResult = ldapPoolConnection.search(searchRequest);
+    LdapUtils.setRequestControls(searchRequest, pageableResult, config);
+    SearchResult searchResult = ldapMonoConnection.search(searchRequest);
     PageResult<ResultType> pageResult = new PageResult<>();
-    List<ResultType> results =
+    pageResult.setResults(
         searchResult.getSearchEntries().stream()
             .map(e -> mapper.mapFromAttributes(e.getAttributes()))
-            .collect(Collectors.toList());
-    pageResult.setResults(results);
+            .collect(Collectors.toList()));
+    LdapUtils.setResponseControls(pageResult, searchResult);
+    pageResult.setNextStart(pageableResult.getFirst() + pageResult.getPageSize());
```

Comment:
offset is never changed so nextStart is not accurate

Reply:
yes because we dont use vlv with vlv there is no cookie only offset